NUService.js

```diff
@@ -419,7 +446,7 @@ export default class NUService extends NUObject {
         } else if (verb === 'POST') {
             return this._connection.makePOSTRequest(requestURL, headers, requestData, cancelToken);
         } else if (verb === 'DELETE') {
-            return this._connection.makeDELETERequest(requestURL, headers, cancelToken);
+            return this._connection.makeDELETERequest(requestURL, headers, requestData, cancelToken);
```

Comment:
Do we need the `requestData` arg if we do not have request body in DELETE calls?

Reply:
Browser's functionality is that when there is no request body payload, the "content" header is automatically removed from the api call. But bulk delete works only when the content type is present. so to make sure that the browser does not remove this header I had to add an empty request body